### PR TITLE
Run test on different env via terminal, update Smoke test

### PR DESCRIPTION
**Pull request by Filipv95**
Added config files for localhost, QA, and PROD ENV.
We are now able to run tests on different ENVs from the terminal without changing branches.
Updated Smoke test, the idea is for the test to check all guilds available in the selected network,  but it is still failing on `check proposal page step` that will be looked at in the new issue that I will create.

### CODE REVIEW

**Comment by Kenny-Gin1** [comment]
Tests are failing. @Filipv95 I think you need to update your `pnpm-lockfile`. Try running `pnpm i` again: https://github.com/pnpm/pnpm/issues/2807

**Comment by Filipv95** [comment]
Hey @Kenny-Gin1 I've pushed the changed `pnpm-lock` file, hopefully that will resolve the issues, just to mention here as well Smoke test is currently failing do some issues I will get to that in another issues.

File: README.md [hunk 1]
```diff
@@ -105,4 +105,23 @@ Running in separate terminals:
...
+# DAVI Monorepo QA tests
+
+## How to setup
+
+1. Install dependencies `pnpm i`
+2. Make an `.env` file in the `davi` folder (see `.env.qa.example`)
+
+### How to run test
+
+To run test just type in terminal `yarn test:testName`
```
Comment (Kenny-Gin1) [inline README.md:119]: Hm, I think we should stick to using `pnpm` as our package manager. I am not sure if there are any conflicts that could arise with having two separate package managers (`yarn` and `pnpm`). 

So basically test the same way you have but use `pnpm` instead :). https://pnpm.io/cli/test

File: apps/davi/cypress/integration/smoke/SmokeTest.spec.ts [hunk 2]
```diff
@@ -4,17 +4,16 @@
 
 /// <reference types="cypress" />
 
-import configDevelopmentJSON from '../../config/development.json';
 import LandingPage from '../../support/pageObjects/LandingPage';
 import AnyGuildPage from '../../support/pageObjects/AnyGuildPage';
 import CreateProposalPage from '../../support/pageObjects/CreateProposalPage';
 import { clickAnywhereToClose, ethereumNetworkGuilds, gnosisNetworkGuilds } from '../../utils';
-
+let url = Cypress.config().baseUrl;
```
Comment (Kenny-Gin1) [inline apps/davi/cypress/integration/smoke/SmokeTest.spec.ts:11]: `const url` since we are not reassigning the url.

**Review by Kenny-Gin1 — CHANGES_REQUESTED** [review]
Nice! Just some small comments!

**Comment by Filipv95** [comment]
@Kenny-Gin1 I've made requested changes, also edited readme a bit, hopefully it's more understandable now, you can check when you have time.

**Review by Kenny-Gin1 — APPROVED** [review]
LGTM! 👍

**Pull request merged**